content/doc/developer/FIPS-140/index.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: developerchapter
+summary: FIPS-140 for developers
+wip: true
+title: FIPS-140 for developers
+references:
+- url: https://github.com/jenkinsci/jep/blob/master/jep/237/README.adoc
+  title: JEP-237 FIPS-140 support for Jenkins
+- url: hhttps://csrc.nist.gov/pubs/fips/140-2/upd2/final
+  title: FIPS-140-2 Security Requirements for Cryptographic Modules
+---
+
+In order for the code in Jenkins and plugins to be compliant with FIPS-140, sensitive information needs to be handled correctly.
+This document provides a non-exhaustive cheat sheet that can guide developers to write code that is FIPS-140 compliant.
+This applies to all forms of cryptography, regardless if it is encryption/decryption, hashing, or signing.
+[IMPORTANT]
```

Comment:
Optional.  I believe that blocks like this are usually preceded by a blank line.  It renders correctly whether the blank line is added or not.
```suggestion

[IMPORTANT]
```